system/jlib/jlzw.cpp

```diff
@@ -1830,6 +1832,8 @@ typedef enum { ICFcreate, ICFread, ICFappend } ICFmode;
 #define COMPRESSEDFILEFLAG (I64C(0xc0528ce99f10da55))
 #define COMPRESSEDFILEBLOCKSIZE (0x10000)
 #define FASTCOMPRESSEDFILEFLAG (I64C(0xc1518de99f10da55))
+#define LZ4COMPRESSEDFILEFLAG (I64C(0xc00000099f10da55))
+#define DENCOMPRESSEDFILEFLAG (I64C(0xc00000198f10db47))
```

Comment:
I don't know either, but guessed also that it was a random pattern.
Probably worth making them a bit more unique (i.e. they currently have quite a few bytes in common) and different to one another.